### DISCUSSION TITLE
Remove paywall?  12ft.io or similar?

### DISCUSSION THREAD

**Discussion by twizzlebizzle** [TrianguloY/URLCheck#221] [Ideas]
Anyway of using urlcheck to remove paywall or redirect a link to a paywall remover like 12ft ladder?

https://www.bloomberg.com/news/articles/2023-04-25/netflix-loses-1-million-users-in-spain-over-password-policing?leadSource=uverify%20wall
  
Here's an example url

**Comment by TrianguloY** [comment]
This service seems to work just by appending 12ft.io to the beginning of the string, so in this case a simple pattern can be added (in the pattern checker module) to do so:

```json
  "12ft.io.": {
    "regex": "^.*$",
    "replacement": "https:\/\/12ft.io\/$0"
  }
```

**Reply by twizzlebizzle** [reply]
Excellent thanks!  Also managed to take the archive.is from the library for pattern checker.

**Reply by twizzlebizzle** [reply]
Bought you a coffee.  Thanks for the app and the prompt reply.  Much appreciated, love this app/project.

**Reply by TrianguloY** [reply]
Oh wow, thank you 😊 much appreciated.

And glad the pattern works. Btw you can add it to the [wiki](https://github.com/TrianguloY/UrlChecker/wiki/Custom-patterns) (other users may find it interesting) or to the [external repository](https://github.com/Raj9039852537/urlChecker-patternchecker-config).
Or I can add it myself if you prefer, no problem.
